src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/TimeSeriesViewer.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import TimeSeriesViewer from './TimeSeriesViewer';
+import { ITimeSeriesViewerProps } from './TimeSeriesViewer.types';
+import MockAdapter from '../../../../Adapters/MockAdapter';
+import { DataHistoryExplorerContext } from '../../DataHistoryExplorer';
+
+const wrapperStyle = { width: '600px', height: '400px' };
+
+export default {
+    title: 'Components/TimeSeriesViewer',
+    component: TimeSeriesViewer,
+    decorators: [getDefaultStoryDecorator<ITimeSeriesViewerProps>(wrapperStyle)]
```

Comment:
can we also update the stories for the data history explorer composed component to have test data in it? Maybe just seed it with a couple series?

Reply:
Added WithSeries stories